sql/src/test/java/io/crate/integrationtests/TransportSQLActionSingleNodeTest.java

```diff
@@ -322,6 +325,59 @@ public void run() {
         }
     }
 
+    @Test
+    public void testKilledJobLog() throws Exception {
+        execute("create table likes (" +
+                "   event_id string," +
+                "   item_id string" +
+                ") clustered into 1 shards with (number_of_replicas = 0)");
+        ensureYellow();
+        execute("SET GLOBAL stats.enabled = true");
+
+        final String stmt = "insert into likes (event_id, item_id) values (?, ?)";
+        final Object[][] bulkArgs = new Object[500][];
+        for(int i = 0; i < bulkArgs.length; i++) {
+            bulkArgs[i] = new Object[]{"event1", "item1"};
+        }
+        final SettableFuture<SQLBulkResponse> res = SettableFuture.create();
+        Thread insertThread = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                try {
+                    res.set(execute(stmt, bulkArgs));
+                } catch (SQLActionException e) {
+                    res.setException(e);
+                }
+            }
+        });
+        insertThread.start();
+        Thread.sleep(50);
```

Comment:
this is flaky, do a loop until you get a result, probably start the insert thread upon first iteration